byron/ledger/executable-spec/test/Ledger/Delegation/Properties.hs

```diff
@@ -167,16 +176,22 @@ instance STS DBLOCK where
 
   initialRules
     = [ do
-          IRC (env) <- judgmentContext
+          IRC env <- judgmentContext
           pure (env, initialDIState)
       ]
 
   transitionRules
     = [ do
           TRC (_, (env, st), dblock) <- judgmentContext
-          env ^. slot < dblock ^. blockSlot ?! NotIncreasingBlockSlot
+          let nextSlot = dblock ^. blockSlot
+          env ^. slot < nextSlot ?! NotIncreasingBlockSlot
           stNext <- trans @DELEG $ TRC (env, st, dblock ^. blockCerts)
-          return (env & slot .~ dblock ^. blockSlot, stNext)
+          -- We fix the number of slots per epoch to 10. This is the same
```

Comment:
It is? I thought that slots per epoch was `21600` in production...

Reply:
Yeah, this is plainly wrong.